pkg/slayers/path/hopfield_spec.gobra

```diff
@@ -16,6 +16,32 @@
 
 package path
 
+ghost const MetaLen = 4
+
 pred (h *HopField) Mem() {
 	acc(h) && h.ConsIngress >= 0 && h.ConsEgress >= 0
 }
+
+ghost 
+decreases
+pure func infoFieldOffset(currINF int) int {
+	return MetaLen + InfoLen * currINF
+}
+
+ghost 
+requires 0 <= currINF
+requires infoFieldOffset(currINF) < len(raw)
+requires acc(&raw[infoFieldOffset(currINF)], _)
+decreases
+pure func ConsDir(raw []byte, currINF int) bool {
+	return raw[infoFieldOffset(currINF)] & 0x1 == 0x1
+}
+
+ghost 
+requires 0 <= currINF
+requires infoFieldOffset(currINF) < len(raw)
+requires acc(&raw[infoFieldOffset(currINF)], _)
+decreases
+pure func Peer(raw []byte, currINF int) bool {
+	return raw[infoFieldOffset(currINF)] & 0x2 == 0x2
+}
```

Comment:
are we redefining these functions in the `router` package? If so, let's drop one copy of each